client/src/components/Header/components/nav-links.tsx

```diff
@@ -462,6 +481,39 @@ export class NavLinks extends Component<NavLinksProps, {}> {
               >
                 {t('buttons.sign-out')}
               </a>
+            </li> */}
+            <li>
+              <button
+                className='nav-link nav-link-signout'
+                onClick={() =>
+                  (this.handleSignOutModal = !this.handleSignOutModal)
```

Comment:
as far as i can see `this.handleSignOutModal` is a function , currently you are passing a boolean value instead

Reply:
````suggestion````
onClick={() => this.setState({showModal: true})}